linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -47,6 +47,7 @@ class HttpInitializer extends ProtocolInitializer.Simple {
       // ensure the client-stack framing filter is placed below the stats filter
       // so that any malframed responses it fails are counted as errors
       .insertAfter(FailureAccrualFactory.role, FramingFilter.clientModule)
+      .insertAfter(FailureAccrualFactory.role, RequestEvaluator.module)
```

Comment:
I just noticed that if you try to use the request evaluator on an address that Linkerd cannot connect to, the connection fails before getting to the evaluator:

```
$ curl localhost:4140 -H "Host: foo" -H "l5d-req-evaluate: 1"
connection timed out: /1.2.3.4:8888 at remote address: /1.2.3.4:8888. Remote Info: Not Available⏎
```

Perhaps if we move the Request higher the stack (above connection establishment) but below the load balancer, this will work?  Perhaps `.insertAfter(LoadBalancerFactory.role)`?